### PR TITLE
Update to v2020.2.16

### PR DESCRIPTION
**Pull request by dmgav**

### CODE REVIEW

File: recipe/meta.yaml [hunk 1]
```diff
@@ -11,21 +11,22 @@ source:
...
+  number: 0
   script: "{{ PYTHON }} -m pip install . --no-deps --ignore-installed -vvv"
   skip: True  # [py<36]
 
 requirements:
   build:
     - {{ compiler('c') }}
   host:
-    - python
+    - python >=3.6
```
Comment (mrakitin) [inline recipe/meta.yaml:22]: Normally it's controlled by the `skip: True  # [py<36]` option (above) for the architecture-dependent packages.

File: .ci_support/osx_.yaml [hunk 2]
```diff
@@ -3,7 +3,7 @@ MACOSX_DEPLOYMENT_TARGET:
 c_compiler:
 - clang
 c_compiler_version:
-- '4'
+- '9'
```
Comment (mrakitin) [inline .ci_support/osx_.yaml:6]: In the `defaults` channel those compilers for OSX are not available yet. Please check how we worked it around in the xraylib-feedstock.

**Review by mrakitin — COMMENTED** [review]
Here are a couple of suggestions.

File: recipe/meta.yaml (hunk 1 above)
Comment (mrakitin) [reply recipe/meta.yaml:22]: And it removed a half of the build configurations. I think it should be reverted to have builds for Python 3.6 and 3.7 for each platform (6 in total).

**Comment by mrakitin** [comment]
https://anaconda.org/conda-forge/imagecodecs-lite has to be build in our channel first if we want to use it. Can we use the non-`lite` version instead?

**Comment by dmgav** [comment]
`imagecodecs-lite` is deprecated. The latest version of `tifffile` requires the package `imagecodecs`, which is available from PyPI. So I guess we have to build `imagecodecs`.

**Comment by mrakitin** [comment]
Closing as outdated.

**Pull request closed without merging**